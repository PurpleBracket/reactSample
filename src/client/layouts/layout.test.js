/* global describe test expect */
import React from 'react'
import { shallow } from 'enzyme'
import AppTemplate from './app.template'
import LandingTemplate from './landing.template'
import { Provider } from 'react-redux'

describe('App Template', () => {
  test('should match App', () => {
    const wrapper = shallow(<Provider><AppTemplate /></Provider>)
    expect(wrapper).toMatchSnapshot()
  })
})

describe('Landing Template', () => {
  test('should match Landng page', () => {
    const wrapper = shallow(<LandingTemplate />)
    expect(wrapper).toMatchSnapshot()
  })
})
