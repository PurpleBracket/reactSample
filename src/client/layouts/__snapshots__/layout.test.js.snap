// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Template should match App 1`] = `<Connect(withRouter(AppTemplate)) />`;

exports[`Landing Template should match Landng page 1`] = `
<div
  className="landingWrapper"
>
  <TopbarHome />
  <div
    className="container nav-padding"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/style"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/signin"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/signup"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/signout"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/password-reset"
      />
      <Route
        component={[Function]}
      />
    </Switch>
  </div>
  <Component />
</div>
`;
